docs/references/react/add-react-router.mdx

```diff
@@ -0,0 +1,286 @@
+---
+title: Add React Router
+description: Learn how to add React Router to your React application using their Data API router.
+---
+
+
+# Add React Router to your Clerk powered React application
```

Comment:
Ahh I see the more informative title here - let's make sure our H1 matches our title tag.

Reply:
I was concerned this was too long for the sidebar. Thoughts on that, or a shorter version?

Reply:
I meant the title in the frontmatter - SEO title and h1 should match